scripts/core/ui/components/content-create-dropdown/more-templates.tsx

```diff
@@ -41,7 +41,12 @@ export class MoreTemplates extends React.PureComponent<IProps, IState> {
             deskCriteria.push({[templateDesks]: {$in: [currentDeskId]}});
         }
 
-        const criteria = {$or: [{$or: deskCriteria}, {user: sdApi.user.getCurrentUserId()}]};
+        const criteria = {
```

Comment:
can you use `ISuperdeskQuery` as [here](https://github.com/superdesk/superdesk-client-core/blob/authoring-react-post-broadcasting/scripts/api/article-send.tsx#L20)?

the logic also seems off inside `criteria`. I'd expect it to be
```$or: [deskCriteria, {user: sdApi.user.getCurrentUserId()}]```
why double $ors?

Reply:
I can't use `ISuperdeskQuery`, since we're working on the `where` filter which we don't do anything for in the `ISuperdeskQuery` or `toElasticQuery`. About the `or` - is because `deskCriteria` is an array of options.